### ISSUE TITLE
Sensor refresh received un-expected SSAPI error

### ISSUE THREAD

**Issue by idealdreams** [homebridge-simplisafe3/homebridge-simplisafe3#163]
### Describe The Bug:
Logs show the below error at sporadic times throughout the day:

[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.

No usability issues noted with the plugin despite the errors. All devices within the plugin (doorbell camera, in-house camera, multiple door, water, and CO sensors) work as expected with no erroneous behavior. 

### To Reproduce:
Run the latest version of this plugin. 

### Expected behavior:
No errors during normal use.  

### Environment:

* **Plugin version**: v1.6.13
* **Node.js Version**: v.12.16.1
* **NPM Version**: v6.14.9
* **Homebridge Version**: v1.2.4
* **Operating System**: Raspbian

### Homebridge Config:

```json
{
            "name": "SimpliSafe",
            "auth": {
                "username": "",
                "password": ""
            },
            "cameras": true,
            "debug": false,
            "sensorRefresh": 15,
            "persistAccessories": true,
            "resetSimpliSafeId": false,
            "cameraOptions": {
                "enableHwaccelRpi": false
            },
            "platform": "homebridge-simplisafe3.SimpliSafe 3"
        }
```

### Screenshots:
Not applicable. 

### Logs:

[Link to logs]()
```
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 8:58:06 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
```

**Comment by shamoon** [comment]
Hi @idealdreams , appreciate the report but these aren't really a bug, per se but I can see how the wording might make it seem so. I found that the SS API is just insanely unreliable and is constantly throwing errors, I pasted just a sampling of my own below. So the wording of this log output is just trying to differentiate between errors the plugin "expects" including `SettingsInProgress` and `GatewayTimeout` errors (which are also insanely common) and it ignores vs other errors. Perhaps it should just log the error and make it clearer that it is SimpliSafe's "fault" or arguably they could just never be logged but when the errors happen the plugin will be unable to update a sensor status so it seemed worth reporting. If you're curious you can [enable `debug`](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3#debug) to get the full error content, e.g.:

```
[12/7/2020, 7:40:14 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 7:40:14 PM] [SimpliSafe] {
  error: 'UNKNOWN_CODE_PLEASE_REPORT',
  error_description: 'UNKNOWN_CODE_PLEASE_REPORT: Max connect timeout reached while reaching hostgroup 10 after 10000ms'
}
[12/7/2020, 7:40:14 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 7:40:14 PM] [SimpliSafe] {
  error: 'UNKNOWN_CODE_PLEASE_REPORT',
  error_description: 'UNKNOWN_CODE_PLEASE_REPORT: Max connect timeout reached while reaching hostgroup 10 after 10000ms'
}
[12/7/2020, 7:40:15 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 7:40:15 PM] [SimpliSafe] {
  error: 'UNKNOWN_CODE_PLEASE_REPORT',
  error_description: 'UNKNOWN_CODE_PLEASE_REPORT: Max connect timeout reached while reaching hostgroup 10 after 10000ms'
}
[12/7/2020, 7:41:35 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 7:41:35 PM] [SimpliSafe] <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
</body>
</html>

[12/7/2020, 7:41:45 PM] [SimpliSafe] Sensor refresh received un-expected SSAPI error.
[12/7/2020, 7:41:45 PM] [SimpliSafe] <html>
<head><title>503 Service Temporarily Unavailable</title></head>
<body>
<center><h1>503 Service Temporarily Unavailable</h1></center>
</body>
</html>
```

Perhaps that was more explanation than you wanted 😄 but again, not really a "bug".

**Comment by phantomsoul1** [comment]
Can you change the way this message is output in the log so that it looks less like a plugin or plugin-configuration error? When the error is just dumped in the log like this, it makes it look like the plugin is not interacting with the API correctly, either by its code or by configuration.

**Comment by shamoon** [comment]
Sure, this seems to be generating some confusion, I released a new version 1.6.14 that should hopefully improve this
